cms/server/admin/templates/participation.html

```diff
@@ -37,125 +37,18 @@
-            {% if sr.compilation_outcome == "fail" %}
-            Compilation failed
-            <div id="evaluation_{{ s.id }}" style="display: none;">
-            {% elif not sr.evaluated() %}
-            Evaluating...
-            <div id="evaluation_{{ s.id }}" style="display: none;">
-            {% elif sr.scored() %}
-              {% try %}
-                {% set max_score = score_type.max_score %}
-              {% except %}
-                {% set max_score = "[Cannot get score type - see logs]" %}
-              {% end %}
-            Evaluated ({{ sr.score }} / {{ max_score }})
-            <div  id="evaluation_{{ s.id }}" class="score_details" style="display: none;">
-              {% raw score_type.get_html_details(sr.score_details) %}
-            {% else %}
-            Evaluated
-            <div id="evaluation_{{ s.id }}" style="display: none;">
-              {% if sr.evaluated() %}
-              <h3>Testcases</h3>
-              <table class="nested bordered">
-                <thead>
-                  <tr>
-                    <th>Outcome</th>
-                    <th></th>
-                    <th>Details</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  {% for idx, ev in sorted((ev.codename, ev) for ev in sr.evaluations) %}
-                  <tr>
-                    <td>{{ ev.outcome }}</td>
-                    {% if s.token is not None or ev.testcase.public %}
-                    <td style="align: center;">&bullet;</td>
-                    {% else %}
-                    <td></td>
-                    {% end %}
-                    <td>{{ ev.text }}</td>
-                  </tr>
-                  {% end %}
-                </tbody>
-              </table>
-              {% end %}
-            {% end %}
-              <h3>Compilation output</h3>{% comment TODO: trim long outputs and add facility to see raw %}
-              <pre>{% if sr.compilation_text is not None %}{{ escape(sr.compilation_text) }}{% end %}</pre>
-            </div>
-          </div>
-
-          {% end %}
-        </td>
-        <td>
-          {% for filename, sub_file in s.files.items() %}
-          <a href="javascript:void(0);" onclick="utils.show_file('{{ filename.replace("%l", s.language) if s.language else filename }}','{{ url_root }}/submission_file/{{ sub_file.id }}')">{{ filename.replace("%l", s.language) if s.language else filename }}<br/>
-          {% end %}
-        </td>
-        <td>
-          {% if s.token is None %}
-          No
-          {% else %}
-          Yes
-          {% end %}
-        </td>
-        <td>
-          {{ s.short_comment }}
-        </td>
-        <td>
-          {% set reevaluation_par_name = "submission" %}
-          {% set reevaluation_par_value = s.id %}
-          {% set reevaluation_par_dataset_id = None %}
-          {% include fragments/reevaluation_buttons.html %}
-        </td>
-      </tr>
-      {% end %}
-    </tbody>
-  </table>
```

Comment:
I totally agree that this code, that was almost duplicated, had to be removed. The "almost" is that, here, the user column was removed, as it value was constant. Whereas now you reintroduced it. No big harm. I suggest that, to handle this properly, the submission table could also be made a UIModule. (I'm not saying you should do it for this PR, it's a proposal for the future)